blaze/compute/sql.py

```diff
@@ -620,11 +644,24 @@ def compute_up(t, s, **kwargs):
     return select(s).limit(t.n)
 
 
-@dispatch(Label, ClauseElement)
+@dispatch(Label, ColumnElement)
 def compute_up(t, s, **kwargs):
     return s.label(t.label)
 
 
+@dispatch(Label, FromClause)
+def compute_up(t, s, **kwargs):
+    assert len(s.c) == 1, \
```

Comment:
Yes it would be very strange to get a label expression on something with more than a single column